### PR TITLE
Revert "Enable iframe-inline-styles e2e test"

### PR DESCRIPTION
**Pull request by aristath**
Reverts WordPress/gutenberg#35171

### CODE REVIEW

File: packages/e2e-tests/specs/editor/plugins/iframed-inline-styles.test.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ describe( 'iframed inline styles', () => {
 		await deactivatePlugin( 'gutenberg-test-iframed-inline-styles' );
 	} );
 
-	it( 'should load inline styles in iframe', async () => {
+	it.skip( 'should load inline styles in iframe', async () => {
```
Comment (kevin940726) [inline packages/e2e-tests/specs/editor/plugins/iframed-inline-styles.test.js:36]: Could we add a comment above for why it's skipped? We can link it to https://github.com/WordPress/gutenberg/issues/35172 as well :)

Comment (kevin940726) [reply packages/e2e-tests/specs/editor/plugins/iframed-inline-styles.test.js:36]: We can prioritize it for the [Playwright migration](https://github.com/WordPress/gutenberg/issues/38851) ;)

Comment (aristath) [reply packages/e2e-tests/specs/editor/plugins/iframed-inline-styles.test.js:36]: > Could we add a comment above for why it's skipped? We can link it to https://github.com/WordPress/gutenberg/issues/35172 as well :)

Good idea 👍 Done in https://github.com/WordPress/gutenberg/pull/41369/commits/d862547cb787843c0be2af23f285ee5d4186f802.

**Review by kevin940726 — APPROVED** [review]
Thanks!

**Pull request merged**